plugin.js

```diff
@@ -46,6 +49,12 @@ function plugin (fn, options = {}) {
     fn.default = fn
   }
 
+  // TypeScript support
```

Comment:
This comment is somehow misleading: in order to truly support TS named import we have to modify also typings to add the correct named property, otherwise, TS won't allow the import (in fact it "just" checks the types, not the code).
This is the main reason I am still trying to figure out how to support it 100% (right now I am not even sure it is possible).